kernel/cmdline.cc

```diff
@@ -23,11 +26,60 @@ cmdlineread(mdev*, char *dst, u32 off, u32 n)
   return cc;
 }
 
+// Returns true if param is found in cmdline, false otherwise.
+// If found, writes the value of the first occurence to dst.
+// Expects cmdline to be a space-delimeted list of <param>=<value> pairs.
+static bool
+getvalue(const char* param, char* dst)
+{
+  char parameq[CMDLINE_PARAM+1];
+  char *p, *end;
+  int i;
+
+  // find '<param>=' in cmdline
+  strcpy(parameq, param);
+  end = parameq + strlen(parameq);
+  *end++ = '=';
+  *end = 0;
```

Comment:
I tried using `strcat` at first but I get the error that the function was not declared in this scope

Reply:
Yeah, sv6 doesn't really have string manipulation code since it isn't needed elsewhere. This seems fine